src/components/ManageVaults.tsx

```diff
@@ -58,7 +58,11 @@ const ManageVaults = () => {
     content = (
       <div className="mt-12 flex flex-wrap gap-x-6 gap-y-8 justify-center xl:justify-start xl:px-2">
         {[...vaults.keys()].map(vaultKey => (
-          <VaultSummary key={vaultKey} vaultKey={vaultKey} />
+          <VaultSummary
+            key={vaultKey}
+            vaultKey={vaultKey}
+            displayFunctions={displayFunctions}
```

Comment:
It used to `assert` that `displayFunctions` wasn't null (because the value returned from the atom is nullable), and it was implicitly up to `VaultSummary`'s parent component to not render it if `displayFunctions` was null.  I made it so that the parent component must explicitly pass a non-null `displayFunctions` so that someone wouldn't accidently render a `VaultSummary` in the wrong place without checking for `displayFunctions` first (which could error non-deterministically depending on loading times).

Though I guess instead of asserting it could just show a loading state if it's null... hmm yea that's better, done.